canonical-webteam/react-starter
# Static images

## Done

- Included [file-loader](https://github.com/webpack-contrib/file-loader)
- Updated `webpack.common.js`
- Added Ubuntu circle of friends logo to the example page

## QA

- Pull the branch
- `yarn install && yarn start`
- See that the dev environment has the logo
- `yarn build`
- Open `dist/index.html` in a browser
- See that the logo shows successfully
